ext_conf_template.txt

```diff
@@ -28,6 +28,9 @@ keepBEGroups = 0
 # cat=backend/enable; type=boolean; label=LLL:EXT:ig_ldap_sso_auth/Resources/Private/Language/locallang_db.xlf:settings.enableBESSO
 enableBESSO = 0
 
+# cat=frontend/enable; type=boolean; label=LLL:EXT:ig_ldap_sso_auth/Resources/Private/Language/locallang_db.xlf:settings.BESSOKeepDomainName
+BESSOKeepDomainName = 0
```

Comment:
the option does not stick to the existing conventions in use, it should be something like `keepBESSODomainName` instead (or alike).

Reply:
and category `cat=frontend/enable` is wrong, it should be `backend/enable` instead.